report/sections/Chapter 1/Intro.tex

```diff
@@ -1 +1,53 @@
+
+\section{Initial Problem} 
+\aau's current restructuring of the course already seems like a major improvement over the former structure but as mentioned it still suffers from a few notable problems.
+\begin{itemize}
+	\item No way to revisit discussed solutions
+	\item No way to verify that an implementation is correct
+	\item No way for the entire group to work together on the same code
+	\item No way for the professor to verify how far the students are in the problem solving
+	\begin{itemize}
+		\item Hard to verify that all students have solved the problems
+		\item Hard to judge the skill-level of the students and if the exercises are suitable
+	\end{itemize}
+\end{itemize}
+
+To try and solve these problems we would like to develop a website targeted toward the professor that enables him to gain a better overview during exercise sessions and therefore allows him to improve the quality of the sessions. To facilitate this, we imagine a website that should allow the professor to:
+\begin{itemize}
+	\item The professor should be able to create a syllabus with the exercises that students are expected to solve
+	\begin{itemize}
+		\item The professor should be able to create multiple syllabi where students can self-enrol into the relevant ones for them
+	\end{itemize}
+	\item The professor should be able to track the amount of exercises completed by each enrolled student
+	\begin{itemize}
+		\item The professor should be able to view additional stats such as attempts used 
+	\end{itemize}
+	\item The professor should be able to specify criteria in order to pass each exercise 
+	\begin{itemize}
+		\item The students should be able to verify is their solution lives up to the specified criteria
+		\item Criteria checking should be handled automatically by the system
+	\end{itemize}
+	\item The students should be able to collaborate on the same code
+	\item The students should have access to the discussed solutions later on
+	\item The professor should be able to see each students code
+\end{itemize}
+
+\subsection{MVP}
+The presented features were then prioritized by a group of students to construct a minimal viable product (MVP) that could be tested and used to verify if the presented product would provide anything useful to the course. Since the MVP is meant to be used as a demo product that allows for additional input to be gathered only the minimum number of functionalities needed were included.
+
+\begin{displayquote}
```

Comment:
This comes too late. Does it need to be a website? That is an implementation detail to the solution? It is unlcear why the erify part is needed to solve the problem.